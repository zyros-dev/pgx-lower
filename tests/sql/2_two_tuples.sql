LOAD 'pgx_lower.so';
SET client_min_messages TO WARNING;
SELECT 'hello';
DROP TABLE IF EXISTS test;
CREATE TABLE test(id SERIAL);
INSERT INTO test(id) VALUES (10);
INSERT INTO test(id) VALUES (1);
SELECT * FROM test;
