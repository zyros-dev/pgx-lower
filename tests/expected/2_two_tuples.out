LOAD 'pgx_lower.so';
NOTICE:  Installing custom executor hook...
NOTICE:  Registering custom sigsegv handler!
SET client_min_messages TO WARNING;
SELECT 'hello';
 ?column? 
----------
 hello
(1 row)

DROP TABLE IF EXISTS test;
CREATE TABLE test(id SERIAL);
INSERT INTO test(id) VALUES (10);
INSERT INTO test(id) VALUES (1);
SELECT * FROM test;
 id 
----
 10
  1
(2 rows)
